src/features/ExtensionCommands.ts

```diff
@@ -382,24 +388,45 @@ export class ExtensionCommandsFeature implements IFeature {
         return promise;
     }
 
-    private saveFile(filePath: string): Thenable<EditorOperationResponse> {
+    private async saveFile(saveFileDetails: ISaveFileDetails): Promise<EditorOperationResponse> {
```

Comment:
Did you have to switch this from a Thenable to a Promise to use async/await?

Reply:
Yeah -- as in @SeeminglyScience's snippet. `Thenable` doesn't type check.

Reply:
good to know!